mro/table.py

```diff
@@ -52,7 +56,7 @@ def _get_cursor(cls):
             try:
```

Comment:
I haven't bothered trying to change the current retry logic here to use `tenacity` instead, because the logic is considerably more complicated depending on the exception type.